packages/shared/topology/utils/AlertFilters.ts

```diff
@@ -23,6 +46,10 @@ export const filterRelevantAlerts = (
   if (_.isEqual(resourceModel, NodeModel)) {
     filter = nodeFilter(resourceName);
   }
+  if (_.isEqual(StorageClusterModel, resourceModel)) {
+    filter = (alert: Alert) =>
+      rgwFilter(alert) || noobaaFilter(alert) || cephFilter(alert);
```

Comment:
no need for this here ??
```suggestion
    filter = (alert: Alert) =>
      rgwFilter(alert) || noobaaFilter(alert) || cephFilter(alert) || commonFilter(alert);
```